packages/opentelemetry-node/test/OTEL_SDK_METRICS.test.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const {test} = require('tape');
+const {runTestFixtures} = require('./testutils');
+
+/** @type {import('./testutils').TestFixture[]} */
+const testFixtures = [
+    {
+        name: 'basic scenario with no sampling configutarion involved (all spans sampled)',
+        args: ['./fixtures/use-http-server-metrics.js'],
+        cwd: __dirname,
+        env: {
+            NODE_OPTIONS: '--import=@elastic/opentelemetry-node',
+            OTEL_METRICS_EXPORTER: 'otlp,console',
+            // The default metrics interval is 30s, which makes for a slow test.
+            // However, too low a value runs into possible:
+            //      PeriodicExportingMetricReader: metrics collection errors TimeoutError: Operation timed out.
+            // which can lead to test data surprises.
+            OTEL_METRIC_EXPORT_INTERVAL: '1000',
+            OTEL_METRIC_EXPORT_TIMEOUT: '900',
+        },
+        // verbose: true,
+        checkResult: (t, err, stdout) => {
+            t.error(err);
+        },
+        checkTelemetry: (t, col) => {
+            t.ok(col.metrics.length > 0);
+
+            const spanMetrics = col.metrics.filter((m) =>
+                m.name.startsWith('otel.sdk.span')
+            );
```

Comment:
This can return an empty array (no `otel.sdk.span...` metrics) and still pass.

```suggestion
            const spanMetrics = col.metrics.filter((m) =>
                m.name.startsWith('otel.sdk.span')
            );
            t.ok(spanMetrics.length > 0);
```

Reply:
right, will add the assertion.

Reply:
commit https://github.com/elastic/elastic-otel-node/pull/849/commits/001a8174b47ccfbc55a73e4d65484f8cdddfe88b